functions/InsertBookingSlotFromQueue/project.json

```diff
@@ -0,0 +1,14 @@
+{
+    "frameworks": {
+        "net46":{
+            "dependencies": {
+                "Microsoft.IdentityModel.Clients.ActiveDirectory": "2.16.204221202", 
+                "Microsoft.Azure.KeyVault": "1.0.0",
```

Comment:
Is the booking slot performing encryption ? 
If not then it shouldn't need the KeyVault Libraries.
We're working from a queue, so actually we probably don't need the ADAL Library either (I suspect this is a hangover from a cut and paste of a another fn's project.json.